### ISSUE TITLE
Bug: Can't update Cyberghost servers

### ISSUE THREAD

**Issue by AlyoshaVasilieva** [qdm12/gluetun#1136]
### Is this urgent?

No

### Host OS

Windows 10

### CPU arch

x86_64

### VPN service provider

Cyberghost

### What are you using to run the container

docker-compose

### What is the version of Gluetun

v3.31

### What's the problem 🤔

Cyberghost servers cannot be updated. Neither the manual method nor UPDATER_PERIOD and waiting works.

Some Cyberghost server entries are old enough that you can't auth to them because the IPs are wrong.

I can update Mullvad servers, so I think there's a bug.

### Share your logs

```plain text
docker run -it --rm -v C:\Temp\cyberghost\update:/gluetun qmcgaw/gluetun:v3.31 update -enduser -providers cyberghost
2022-09-07T10:04:16Z INFO merging by most recent 11434 hardcoded servers and 11498 servers read from /gluetun/servers.json
2022-09-07T10:04:16Z INFO Using mullvad servers from file which are 137 days more recent
2022-09-07T10:04:16Z INFO updating Cyberghost servers...
2022-09-07T10:04:36Z ERROR cannot update server information: cannot get servers: not enough servers found: 0 and expected at least 147
2022-09-07T10:04:36Z INFO Shutdown successful
```

**Comment by AlyoshaVasilieva** [comment]
Also I don't think this is right:

https://github.com/qdm12/gluetun/blob/7506625f40a4d9cb213ef21ca871a19164d58167/internal/provider/cyberghost/updater/constants.go#L5-L16

I wrote a program to resolve all possible UDP servers to IPs and only `87-1` and `87-8` ever have records, `94-1` and `95-1` don't seem to exist. American and Asian servers are listed under `87-1`. (Didn't check TCP records)

**Comment by qdm12** [comment]
```
Unable to find image 'qmcgaw/gluetun:v3.31.0' locally
v3.31.0: Pulling from qmcgaw/gluetun
213ec9aee27d: Already exists 
dabbdf2445db: Pull complete 
e52a837c2fad: Pull complete 
Digest: sha256:9ef4853dab38fb829d481283e9f154243b61e54e3fa7973fa9bfb966d08f9f19
Status: Downloaded newer image for qmcgaw/gluetun:v3.31.0
2022-09-08T12:22:16Z INFO creating /gluetun/servers.json with 11434 hardcoded servers
2022-09-08T12:22:16Z INFO updating Cyberghost servers...
```

does succeed for me 🤔 Maybe your network/dns didn't work?

Regarding the outdated groups, I just fixed it in aba5ca4536b803c8eb41c654baf50ef5f4240ae1 thanks for the heads up on that one! It should lower the amount of 'brute-force' dns requests to find their servers. I also updated the servers information built in at the same time.
